src/components/WebexAvatar/README.md

```diff
@@ -1 +1,7 @@
-Webex Avatar Component!
+# Webex Avatar Component
+Webex avatar component displays the avatar of a *Webex Teams* user with a given `personID`.
+
+## How to use
+```js
+  <WebexAvatar personID="PersonID" />
```

Comment:
Should have the adapter defined here along with some details on methods the adapter needs and the properties the component expects to get from the adapter.